deployments/base/weth/configuration.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Compound WETH",
+  "symbol": "cWETHv3",
+  "baseToken": "WETH",
+  "baseTokenAddress": "0x4200000000000000000000000000000000000006",
+  "borrowMin": "1e0",
```

Comment:
this is intentional, right?

Reply:
Yeah. It could also be 0, but I think 1 wei makes more sense.